diff --git a/run_baseline_PPO.py b/run_baseline_PPO.py
index 6684148..5ff5967 100644
--- a/run_baseline_PPO.py
+++ b/run_baseline_PPO.py
@@ -62,6 +62,10 @@ if __name__ == "__main__":
     learn_steps = 32
     sess_id = str(uuid.uuid4())[:8]
 
+    reward_scale = 1
+    explore_weight = 0.5
+    battle_weight = 4
+
     sess_path = Path(f'Sessions/{algorithm}_Session_{current_datetime_str}_{sess_id}_env2_1')
     # sess_path = Path(f'Sessions/PPO_Session_0307161249_7602f77b_env2_2')
     print(sess_path)
