diff --git a/run_baseline_v2.py b/run_baseline_v2.py
index 9f490ab..575b292 100644
--- a/run_baseline_v2.py
+++ b/run_baseline_v2.py
@@ -83,7 +83,7 @@ if __name__ == "__main__":
             'headless': True, 'save_final_state': True, 'early_stop': False,
             'action_freq': 24, 'init_state': 'has_pokedex_nballs.state', 'max_steps': ep_length, 
             'print_rewards': True, 'save_video': False, 'fast_video': True, 'session_path': sess_path,
-            'gb_path': 'PokemonRed.gb', 'debug': False, 'reward_scale': 1.2, 'explore_weight': 1, 'battle_weight': 5,
+            'gb_path': 'PokemonRed.gb', 'debug': False, 'reward_scale': 1, 'explore_weight': 0.5, 'battle_weight': 4,
             'use_screen_explore': True, 'extra_buttons': False, 'sim_frame_dist': 2_000_000.0,
         }
     
