diff --git a/run_baseline_v2.py b/run_baseline_v2.py
index 8fd0c06..e57205e 100644
--- a/run_baseline_v2.py
+++ b/run_baseline_v2.py
@@ -66,7 +66,6 @@ if __name__ == "__main__":
     # sess_path = Path(f'Sessions/PPO_Session_0307161249_7602f77b_env2_2')
     print(sess_path)
 
-
     num_cpu = 14  #! cannot go any higher than 12 <- also crashes after 3-4 hours
     episode_length_per_cpu = 1250 #? each episode will be 1250 steps long 
     ep_length = num_cpu * episode_length_per_cpu #? EPISODE LENGTH WILL BE 17,500
